saleor/graphql/webhook/mutations.py

```diff
@@ -306,3 +316,80 @@ def perform_mutation(cls, _root, info: ResolveInfo, **data):
         manager = get_plugin_manager_promise(info.context).get()
         manager.event_delivery_retry(delivery)
         return EventDeliveryRetry(delivery=delivery)
+
+
+class WebhookDryRun(BaseMutation):
+    payload = JSONString(
+        description="JSON payload, that would be sent out to webhook's target URL."
+    )
+
+    class Arguments:
+        query = graphene.String(
+            description="The subscription query that defines the webhook event and its "
+            "payload.",
+            required=True,
+        )
+        object_id = graphene.ID(
+            description="The ID of an object to serialize.", required=True
+        )
+
+    class Meta:
+        description = "Performs a dry run of a webhook event."
+        permissions = (AppPermission.MANAGE_APPS,)
+        error_type_class = WebhookDryRunError
+
+    @classmethod
+    def validate_input(cls, info, **data):
+        query = data.get("query")
+        object_id = data.get("object_id")
+
+        event_type = get_event_type_from_subscription(query)
```

Comment:
This function returns the first event_type that is in the query, right? Maybe we should mention this in the description of the mutation, as user can provide a multiple events like `orderCreated`, `orderFullyPaid` etc in single query

Reply:
Or maybe we should raise an exception when we find multiple events? 🤔

Reply:
Can you give me an example of the query with multiple events you think about?

Reply:
Currently, a single event (the first if multiple provided) will be supported